### ISSUE TITLE
Results are not colorblind friendly

### ISSUE THREAD

**Issue by Pound-Hash** [StateOfJS/Monorepo#80]
I'm moderately green-weak colorblind.

Four of the following categories are indistinguishable to me:

![20220219_17h37m51s_grim](https://user-images.githubusercontent.com/51193876/154825063-37829ebf-dbe8-4977-9741-404ff6d5839a.jpeg)

- Back-end & B

**Comment by SachaG** [comment]
Oh that sucks, thanks for letting me know.

**Comment by hta218** [comment]
That's exactly what I wanna say @Pound-Hash 

The colors are a bit confused to color-blinded ppl (like me)

Please make the contrast higher @SachaG thanks a lots!

**Comment by SachaG** [comment]
Is it better now? https://2021.stateofjs.com/en-US/libraries

**Comment by Pound-Hash** [comment]
> Is it better now? https://2021.stateofjs.com/en-US/libraries

Yes, it is. Monorepo and Back-end are a little too similar. I can differentiate them, but not by much. Some other users will likely have trouble with these.

Coincidentally, I've spent these past few weeks working on colorblind themes for Kitty terminal. Today I may have finally alighted on a decent setup. What works for me is spacing the hues out by 60 deg, saturating them fully (except for red because it bleeds on me - I'm green-weak), and adjusting the lightness values until they harmonize with the dark background.

But you're not working with text in terminals or text editors, so the strategy doesn't apply entirely.

**Comment by SachaG** [comment]
Yeah it was hard to find colors that are distinct enough while also maintaining good contrast… I'm not sure what the solution is, but in this specific case you can also hover over the legend items and all items of that category will be highlighted. Maybe in the future we can also use patterns or some other non-color way of indicating categories.

**Comment by tim-mehlhorn** [comment]
Exploring different shapes, along with the different colors, might help differentiate the data without solely relying on color. 

Here is quick example what just rounding different corners might look like (border-radius seemed the easiest way to make different shape with the existing output)

![Different-Shapes](https://user-images.githubusercontent.com/3484911/181576041-45fda62a-a910-4ed2-8623-945ead09acbe.png)
